app/views/home/slice/slice.html.haml

```diff
@@ -0,0 +1,66 @@
+- if params[:pop].eql?("true")
+  - pop = true
+- unless @errors.nil?
+  %div{style: "color:red;margin-bottom:1em;"}
+    = t("home.slice.error_on_form")
+    %ul
+      - @errors.each do |error|
+        %li= error
+= form_tag('/slice') do
+  = hidden_field_tag 'sim_submit', 'true'
+  = hidden_field_tag("pop", pop)
+  .d-flex.justify-content-center.m-3
+    .feedback-form.p-4{style: "max-width: 600px; width: 100%;"}
+      .feedback-title-container.text-center.mb-4
+        %h2.feedback-title
+          = t("home.slice.suggest_a_slice")
+        %hr#feedback-title-line.mb-4/
+
+      .form-group.mb-3{style: "max-width: 500px; margin: 0 auto;"}
+        %p.feedback-input-title
+          = t("home.slice.email")
+          %font{color: "red"} *
+        - if session[:user]
+          = hidden_field_tag "email", session[:user].email
+          %input.feedback-input.w-100{name: "email", type: "email", value: session[:user].email, disabled: true}/
```

Comment:
you don't sue the input components

Reply:
use `render Input::TextInputComponent.new(label: label, name: "name", placeholder: placeholder, error_message: error_message, helper_text: helper_text)`